nautobot_floor_plan/filters.py

```diff
@@ -15,13 +16,13 @@ class FloorPlanFilterSet(NautobotFilterSet):
     q = SearchFilter(
         filter_predicates={
             "location__name": "icontains",
-            "location__slug": "icontains",
         },
     )
     location = NaturalKeyOrPKMultipleChoiceFilter(
         queryset=Location.objects.all(),
-        label="Location (slug or ID)",
+        label="Location (name or ID)",
     )
+    # TBD: Verify it's necessary
     tag = TagFilter()
```

Comment:
```suggestion
```

Add `"tags"` to fields in Meta below. Same for `FloorPlanTileFilterSet`.

Reply:
When I removed this line and added it to Meta.fields, tests were complaining tag is missing in the model, so keeping it as is, just removed TBDs.

Reply:
Fixed now by removing `tag` filter and replacing with `tags`.